src/main/java/org/opensearch/ad/model/DetectorProfile.java

```diff
@@ -50,7 +49,7 @@
     private InitProgressProfile initProgress;
     private Long totalEntities;
     private Long activeEntities;
-    private Map<String, ADTaskProfile> adTaskProfiles;
+    private ADTaskProfile adTaskProfile;
```

Comment:
Can you explain why multiple profiles were passed before?

Reply:
Different way to organize task profile. Previous design treat detector level and entity level task profile as the same `ADTaskProfile`, that's not BWC as the AD 1.0 code is using `ADTaskProfile`. So now I changed it as `ADTaskProfile` to make it BWC with 1.0. And put both detector&entity level profiles in it.

Reply:
Check example https://github.com/opensearch-project/anomaly-detection/pull/170#discussion_r694416515